CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
-## [UNRELEASED] neptune-sklearn 2.1.1
+## [UNRELEASED] neptune-sklearn 2.2.0
```

Comment:
Only features bump the minor - [semantic versioning](https://semver.org/)

Reply:
Bumped the minor because of bumping the python dependency. Can use 2.1.1 if we decide not to bump the dependency.